### DISCUSSION TITLE
Floating edges label misplaced

### DISCUSSION THREAD

**Discussion by activey** [bcakmakoglu/vue-flow#968] [Q&A] [answered]
Hi again ;>

After applying all the changes we've discussed today I'm facing one more issue. Here is the sandbox url: ttps://codesandbox.io/p/sandbox/blue-dawn-zkb7yv

The problem is hard to reproduce but here is the result:

![image](https://github.com/bcakmakoglu/vue-flow/assets/1751891/5879e924-d292-4a73-ba21-6027245e0721)

For each edge there is a custom label renderer that displays the label x and y location, as you can see on the screenshot for node8 the label value stays " x " until you move node8 or any node connected to it. Because of that one of the labels always stays at 0x0 until node is moved.

Any ideas? ;)

**Comment by bcakmakoglu** [comment]
You're missing `position: 'absolute'` on your edge label renderer.

[Here's a working sandbox.](https://codesandbox.io/p/sandbox/trusting-sea-9dyg4i?file=%2Fsrc%2Fcomponents%2FFloatingEdge.vue%3A26%2C28)

**Reply by activey** [reply]
Unfortunately I was still able to reproduce this issue, please see the video attached.

https://github.com/bcakmakoglu/vue-flow/assets/1751891/81671627-fc71-4a0a-862c-42d79788c87c

**Reply by activey** [reply]
I'm sending new sandbox link: https://codesandbox.io/p/sandbox/sleepy-darkness-jhdx1l
It happens when all nodes have starting position of (0,0)
After moving them a bit from initial positions I end up with something like this:

![image](https://github.com/bcakmakoglu/vue-flow/assets/1751891/57080908-5f19-4eb7-a2ff-ed07d6bd9888)

**Reply by activey** [reply]
I've just tried it with react-flow, the effect is the same ;D Both vue-flow and reac-flow are affected.

**Reply by bcakmakoglu** [reply, marked as answer]
It's not library code that isn't working, it's the calculation utils and I currently don't have the time to debug it.
Easiest solution: Don't use nodes that overlap on the same coordinates and don't use 0,0 and it should be fine.

**Reply by activey** [reply]
Yup, I will try to use some random initials, anyway, thanks for all your help so far ;) I will try to contact react-flow devs in order to solve it.
